homeassistant/components/wallbox/sensor.py

```diff
@@ -127,6 +129,16 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
         icon="mdi:ev-station",
         name="Depot Price",
         precision=2,
+        device_class=SensorDeviceClass.MONETARY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    CHARGER_ENERGY_PRICE_KEY: WallboxSensorEntityDescription(
+        key=CHARGER_ENERGY_PRICE_KEY,
+        icon="mdi:ev-station",
+        name="Energy Price",
```

Comment:
What kind of measurement is this price? Is it an aggregate cost eg per time or is it a price per unit of energy?